component/main.jsonnet

```
@@ -177,14 +177,43 @@ local plan = [
     ) else
       [];
 
+  local command(p) =
+    if std.objectHas(p, 'command') then (
+      if std.type(p.command) == 'string' then (
+        [ p.command ]
+      )
+      else (
+        if std.type(p.command) == 'array' then (
+          p.command
+        ) else
+          error 'Field `command` of plan "%(name)s" is not an array nor a string' % p
+      )
+    ) else
+      [];
+
+  local version = (
+    if std.objectHas(p, 'version') then
+      p.version
+    else
+      ''
+  );
```

Setting `version=null` if it's not provided in the input may be the more robust approach, cf. my comment on the component library.

```suggestion
  local version = if std.objectHas(p, 'version') then p.version;
```